example/jupyter/tutorial_xgboost.ipynb

```diff
@@ -0,0 +1,357 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# XGBoost on SQLFlow Tutorial\n",
+    "\n",
+    "This is a tutorial on train/predict XGBoost model in SQLFLow, you can find the design doc from [here](https://github.com/sql-machine-learning/sqlflow/blob/develop/doc/xgboost_on_sqlflow_design.md), in this tutorial you will know how to:\n",
```

Comment:
no need to mention design in a tutorial?

Reply:
"you will know how to" -> learn how to...

Reply:
> no need to mention design in a tutorial?

Yep, already linked the user guide doc.

Reply:
done.